x-pack/plugins/ml/public/alerting/anomaly_detection_alerts_table/render_cell_value.tsx

```diff
@@ -54,22 +49,17 @@ const getRenderValue = (mappedNonEcsValue: any) => {
   return '—';
 };
 
-export const getRenderCellValue = (fieldFormats: FieldFormatsRegistry): GetRenderCellValue => {
+export const getRenderCellValue: RenderCellValue = ({ columnId, data, fieldFormats }) => {
```

Comment:
Do you think we can be explicit with the return type?
```suggestion
export const getRenderCellValue: RenderCellValue = ({ columnId, data, fieldFormats }): GetRenderCellValue | undefined => {
```

Reply:
what's the benefit of that? I actually removed that type from the triggersActionsUI plugin types, as it's no longer a function that takes context and setAlertFlyout that returns a function that takes props that returns jsx, it's just basically a component, function that takes props and returns jsx. Even if that type was changed to just be a component of props of type  EuiDataGridCellValueElementProps & { data: TimelineNonEcsData[] } like it is, I think that type itself is wrong, as data can be falsey in some cases, and typescript is not complaining https://github.com/elastic/kibana/blob/main/x-pack/plugins/triggers_actions_ui/public/application/sections/alerts_table/cells/render_cell_value.tsx#L68